### DISCUSSION TITLE
Request: Exalted vs WoD character sheet

### DISCUSSION THREAD

**Discussion by Nealend86** [JohanFalt/Foundry_WoD20#890] [Q&A] [answered]
Exalted vs WoD is a moderately popular fan supplement for the 20th Anniversary line of World of Darkness products made by Holden Shearer, former game designer for White Wolf and Onyx Path. I was wondering if you'd be willing to include it as a character sheet option. The supplement itself is freely available on Holden's wordpress

**Comment by JohanFalt** [comment]
I'll check this.

**Comment by JohanFalt** [comment]
Will look into this for v4.2.

**Comment by JohanFalt** [comment]
Begun the work today.

**Reply by Nealend86** [reply]
Thank you.

**Comment by JohanFalt** [comment, marked as answer]
Will be present in v4.2
